projects/lwan/request_fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+dict = http.dict
+max_len = 32767
```

Comment:
max_len here won't work for AFL which is one of two fuzzing engine we use
just use this code below in fuzz target.

if (size > 32767)
  return 0;

Reply:
Done.